### ISSUE TITLE
<ibm-combo-box> Accessibility Issue

### ISSUE THREAD

**Issue by vigneswaran000** [IBM/carbon-components-angular#1876]
- An element with WAI-ARIA role 'combobox' does not have the required WAI-ARIA attribute(s): 'aria-expanded'
- The WAI-ARIA attribute 'aria-disabled' is not valid for the element with WAI-ARIA role 'none'

![](https://user-images.githubusercontent.com/3095717/127095567-5666259c-93dc-4a7a-bc55-1269a223f8af.png)

![](https://user-images.githubusercontent.com/3095717/127095571-e2d14c45-8ff2-4483-aae8-3efea92a947a.png)

![](https://user-images.githubusercontent.com/3095717/127095663-4e0c0f86-2c9b-47ac-86a4-bb03e345a731.png)
